docs/vaahcms-2/backend/roles.md

```diff
@@ -0,0 +1,238 @@
+# Roles
+
+[[toc]]
+
+## Purpose
+A role is a group of permissions that allow a user who has been given the role to carry out specific tasks. For instance, the super administrator, who can be regarded as the application's owner, is permitted to carry out all tasks in the application. He or she can add users, remove or edit, and do other things.
+
+## Features & Demos
+
+#### Default Roles 
+
+VaahCMS gave you a few default roles that it built during application setup using seeds.
+
+| Roles    | Description                                                  |
+| ------------- | ------------------------------------------------------------ |
+| Super Administrator | The application's owner is super administrator. Users with super administrator privileges have full access and control over the data and application. |
+| Administrator | Users who have administration roles has all the permissions access and manage the data. Somebody who has access to all the administration features within the application. |
+| Manager       | Users who have manage roles can assign a role to user. |
+| Editor | Somebody who can publish and manage posts from the `CMS` section including the posts of other users. |
+| Contributor | Somebody who can write and manage their own posts but cannot publish them.|
+| Registered    | Users who have registered roles can access only public pages. |
+| Guest | Users who have guest roles can access only public pages. |
+
+------
+
+
+#### Create New Role
+You can view the video attached below to learn how to create a new role.
+
+<figure class="video_container">
+  <iframe src="https://img-v4.getdemo.dev/screenshot/chrome_qDABg3eqi2.mp4" frameborder="0" allowfullscreen="true" style="width: 100%; aspect-ratio: 16/9;"> </iframe>
+</figure>
+
+
+####  Assign Permission to Roles
+You can view the video attached below to learn how to add permission to a role.
+
+<figure class="video_container">
+  <iframe src="https://img-v4.getdemo.dev/screenshot/chrome_S5yeP8t3QQ.mp4" frameborder="0" allowfullscreen="true" style="width: 100%; aspect-ratio: 16/9;"> </iframe>
+</figure>
+
+####  Assign Roles to User
+You can view the video attached below to learn how to give a role to a user.
+
+<figure class="video_container">
+  <iframe src="https://img-v4.getdemo.dev/screenshot/chrome_EiAGS8V1Y5.mp4" frameborder="0" allowfullscreen="true" style="width: 100%; aspect-ratio: 16/9;"> </iframe>
+</figure>
+
+## Permissions
+
+The following permissions are necessary for role management in order to carry out certain actions.
```

Comment:
@wri-tanmoy-d001 `role` -> `roles`

Reply:
@themodernpk sir,

sir, I'll update the `PR` and fix my mistakes